EDA/LibRAG_EDA.ipynb

```diff
@@ -0,0 +1,2124 @@
+{
```

Comment:
<p>Just to be clear, it's just the metadata file itself that is 6.7GB which doesn't include the metadata, right?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BU-Spark/ml-bpl-rag/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>